setup.cfg

```diff
@@ -27,6 +27,7 @@ exclude = .git,__pycache__,.tox,.venv
 per-file-ignores =
     plugins/*:E402
     roles/*:E402
+select = C, E, F, W, B, B9
```

Comment:
It would be good to have a comment here about the meaning of the items.

Reply:
The default configuration works for us. I removed this line.